python_bindings/tests/end-to-python-end-test/other_two_player_game_test.py

```diff
@@ -0,0 +1,636 @@
+|          .....|   |..|..|   |
+|          ......\ /...|...\ /.
+|0         ..8----0----1....3..
+|          ..|.........|.../ \.
+|          ..|.........|..|   |
+|          ..|....@....|..|   |
+|          ..|.........|...\ /
+|-1        ..B----6----5....7..
+|          ..|..........!../ \.
+|          ..!............| ! |
+|
+|
+|-2
+|
+|
+"""
+
+
+def check_eleventh_turn(engine: GameEngine, game, game_id) -> (int, SerializedGame):
+    turn_params = TurnParams(
+        pos=Position(x=-1, y=-1),
+        tile=game.current_tile,
+        meepleType=MeepleType.NormalMeeple,
+        side=Side.Bottom,
+        featureType=FeatureType.Road,
+    )
+
+    game_id, game = make_turn(engine, game, game_id, turn_params)
+    check_points(game, [8, 12])
+
+    return game_id, game
+
+
+"""
+// Player2 places Straight road with a meeple on a road
+|				  0	   1    2    3
+|
+|
+|               |   |.......!.......
+|               .\ /............[ ].
+|1              ..2....4----9---[A].
+|               ./ \...|.../ \..[@].
+|               |   |..|..|   |.....
+|          .....|   |..|..|   |
+|          ......\ /...|...\ /.
+|0         ..8----0----1....3..
+|          ..|.........|.../ \.
+|          ..|.........|..|   |
+|          ..|....@....|..|   |
+|          ..|.........|...\ /
+|-1        ..B----6----5....7..
+|          ..|..........!../ \.
+|          ..!............| ! |
+|               .....
+|               .....
+|-2             --C-@
+|               .....
+|               .....
+"""
+
+
+def check_twelfth_turn(engine: GameEngine, game, game_id) -> (int, SerializedGame):
+    turn_params = TurnParams(
+        pos=Position(x=0, y=-2),
+        tile=tiletemplates.straight_roads(),
+        meepleType=MeepleType.NormalMeeple,
+        side=Side.Right,
+        featureType=FeatureType.Road,
+    )
+
+    game_id, game = make_turn(
+        engine, game, game_id, turn_params, True, {1: 8 + 11, 2: 12 + 4}
+    )
+    check_points(game, [8, 12])
+
+    return game_id, game
+
+
+"""
+player1 scores 11 points in total:
+    - 3*3points for farmer on 9 and 5 tile
+    - 1 point for road on B tile
+    - 1 point for city on 7 tile        
+
+player2 scores 4 points in total:    
+    - 1 point for road on C tile
+    - 3 points for monastery on A tile    
+
+|				  0	   1    2    3
+|
+|
+|               |   |.......!.......
+|               .\ /............[ ].
+|1              ..2....4----9---[A].
+|               ./ \...|.../ \..[@].
+|               |   |..|..|   |.....
+|          .....|   |..|..|   |
+|          ......\ /...|...\ /.
+|0         ..8----0----1....3..
+|          ..|.........|.../ \.
+|          ..|.........|..|   |
+|          ..|....@....|..|   |
+|          ..|.........|...\ /
+|-1        ..B----6----5....7..
+|          ..|..........!../ \.
+|          ..!............| ! |
+|               .....
+|               .....
+|-2             --C-@
+|               .....
+|               .....
+"""
```

Comment:
Shouldn't there be a finalize after this? (Like in all_tiles_test.py)

Reply:
finalize is obsolete